packages/ui/theme-builder/app/theme-builder.tsx

```diff
@@ -37,6 +37,7 @@ export function ThemeBuilder({ children }: { children: React.ReactNode }) {
   const [spacingUnit, setSpacingUnit] = useState(spacingUnitDefault);
   const [fontSize, setFontSize] = useState(fontSizeDefault);
   const [devMode, setDevMode] = useState('on');
+  const [animations, setAnimations] = useState('on');
```

Comment:
```suggestion
  const [animations, setAnimations] = useState<'on' | 'off'>('on');
```

Reply:
yeah, I have been keeping these simple for now and not worrying to much about types. There will likely be a refactor of the theme-builder once we ready to go live with it.